supertinycron/README.md

```diff
@@ -0,0 +1,261 @@
+
+supertinycron
+=============
+
+A very small replacement for cron. Particularly useful within containers and for distributing cron tasks alongside a project as a standalone file.
+
+Installing
+----------
+
+```bash
+make install
```

Comment:
That's probably going to be pretty confusing to the users of this component in IDF... They don't need to run `make install`, neither will the `Usage` part work the same way. I understand you have probably copied the original Readme file. Can we instead provide a readme file which will actually going to be useful to IDF users? For example, how they can use this component in their IDF project — how to add the component, which APIs to call, and so on.

Reply:
Sure, i have added simple README and example to show how to use in IDF project